src/devices/Media/VideoDevice/Devices/UnixVideoDevice.cs

```diff
@@ -488,7 +539,7 @@ private void Close()
         /// <param name="request">V4L2 request value</param>
         /// <param name="struct">The struct need to be read or set</param>
         /// <returns>The ioctl result</returns>
-        private int V4l2Struct<T>(VideoSettings request, ref T @struct)
+        private int V4l2Struct<T>(int request, ref T @struct)
```

Comment:
Why is this changed? Wasn't enum better?

Reply:
The enum was using pre-calculated values for the request instead of getting the correct values based on the struct size.